aequilibrae/paths/route_choice.py

```diff
@@ -75,7 +75,7 @@ def set_choice_set_generation(self, /, algorithm: str, **kwargs) -> None:
 
         Setting the parameters for the route choice:
 
-        `beta`, `theta`, and `seed` are BFSLE specific parameters.
+        `beta` and `seed` are BFSLE specific parameters.
```

Comment:
`beta` is the utility parameter on path overlap for probability calculations when using path size logit